clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/settings/views/audio/AudioSettingsViewMediatorBase.as

```diff
@@ -0,0 +1,10 @@
+package org.bigbluebutton.lib.settings.views.audio {
+	
+	import robotlegs.bender.bundles.mvcs.Mediator;
+	
+	public class AudioSettingsViewMediatorBase extends Mediator {
+		
+		[Inject]
+		public var view:AudioSettingsViewBase;
```

Comment:
Common parts moved from `AudioSettingsViewMediatorAIR to AudioSettingsViewMediatorBase`